app/src/components/v-form/form-field-raw-editor.vue

```diff
@@ -87,6 +91,7 @@ const setRawValue = () => {
 					:disabled="disabled"
 					language="plaintext"
 					:placeholder="t('enter_raw_value')"
+					:is-big-integer="props.field.type === 'bigInteger'"
```

Comment:
Let's try to work with BigInts as strings where possible so we limit the amount of "is big int" props added to where they are strictly required. I think we can drop a bunch of the "if big int then" logic if `getJSType` were to return `string` for `bigInt` instead of `number`.